src/amo/homeHeroItems.js

```diff
@@ -0,0 +1,188 @@
+import type { I18nType } from 'core/types/i18n';
+
+export function getHeroItems(i18n: I18nType) {
```

Comment:
Can you put this in the `HomeHeroBanner` file directly?

Reply:
I'd add all this code in `sections()` in `HomeHeroBanner`. Maybe you can use your `heroes` variable for that!